gradle.properties

```diff
@@ -1,6 +1,4 @@
 org.gradle.daemon=true
 org.gradle.parallel=true
 org.gradle.configureondemand=true
-org.gradle.jvmargs=-Xmx3g -XX:MaxPermSize=2048m -XX:+HeapDumpOnOutOfMemoryError -Dfile.encoding=UTF-8
-
-includeSample=true
```

Comment:
I don't think you can remove this? If you do, you have to delete the check in `settings.gradle.kts` as well, I guess.